services/web/client/source/class/osparc/component/widget/logger/LoggerView.js

```diff
@@ -291,7 +291,7 @@ qx.Class.define("osparc.component.widget.logger.LoggerView", {
 
     __downloadLogs: function() {
       const logs = this.__getLogsString();
-      osparc.utils.Utils.downloadContent("data:text/json;charset=utf-8," + logs, "logs.json");
+      osparc.utils.Utils.downloadContent("data:text/plain;charset=utf-8," + logs, "logs.txt");
```

Comment:
i was about to write you an issue about this 👍

Reply:
MINOR: I would use the ``.log`` extension which is standard for logs (e.g. vscode automatically highlights logs)

Reply:
Done